documentation/tutorials/1-overview.md

```diff
@@ -0,0 +1,23 @@
+# Charmed OpenSearch tutorial
```

Comment:
Copy the content mentioned in this file: https://docs.google.com/document/d/18scYgleIOCk3Z1cMgbfUQVIN6GhuVb-eSJG8cPG11dA/edit#